podcasts/Sharing/ShareImageView.swift

```diff
@@ -24,46 +24,74 @@ enum ShareImageStyle: CaseIterable {
             return "small"
         }
     }
+
+    var videoSize: CGSize {
+        switch self {
+        case .large:
+            CGSize(width: 292, height: 438)
+        case .medium:
+            CGSize(width: 292, height: 293)
+        case .small:
+            CGSize(width: 324, height: 169)
+        }
+    }
 }
 
 struct ShareImageView: View {
 
     let info: ShareImageInfo
     let style: ShareImageStyle
 
+    @Binding var angle: Double
+
     var body: some View {
         ZStack {
             LinearGradient(gradient: info.gradient, startPoint: .top, endPoint: .bottom)
+            let rotationFactor = sin(Angle(degrees: angle).radians)
+            KidneyShape()
```

Comment:
These are likely not the final animations but get us relatively close to the original designs.